troubleshooting-input-type-file-not-working-on-android.md

```diff
@@ -0,0 +1,85 @@
+---
+title: Input Type File Not Working on Android
+description: HTML input element of type file is not working in the Android web view.
+type: troubleshooting
+page_title: HTML File Input Not Responding in the Android Web View
+slug: troubleshooting-input-type-file-not-working-in-android
+position:
+tags: appbuilder, android
+teampulseid:
+ticketid:  1002377,  1020566, 1018666
+pitsid:
+
+---
+
+## Environment
+<table>
+  <tr>
+    <td>Mobile development type</td>
+    <td>Hybrid (Cordova app)</td>	
+  </tr>
+  <tr>
+    <td>Mobile OS</td>
+    <td>Android</td>	
+  </tr>
+  <tr>
+    <td>Mobile OS version</td>
+    <td>4.4 KitKat and above</td>	
+  </tr>  
+</table>
+
+## Description
+Using `<input type="file">`
+doesn't work on some Android devices. In most of the cases, when the **Choose File** button is tapped there is no response from the web view and the user cannot select a file from the device. 
+
+
+## Steps to Reproduce
+1. Create a Cordova application.
+1. Insert the `<input type="file">` element in the `<body>` of the *index.html* file.
+1. Build and deploy the project to an Android 4.4 or higher device.
+1. Try to interact with the **Choose File** button.
+
+## Cause
+This is an external issue in the Google Chromium web view. More about it can be found here:
```

Comment:
We should probably pick some template phrasing such as:

* More information about this is available here
* More information available here
* More information here
* More information (the more information phrase being a link)